Services.Test/Concurrency/PerSecondCounterTest.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
+using Services.Test.helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Services.Test.Concurrency
```

Comment:
nit: Why the concurrency namespace?  ThrottlingLimits = better name?

Reply:
we had a preexisting "concurrency" namespace which seemed good enough. I'm ok moving the code into a dedicated space if you say, however I think it's still in the same space of problems

Reply:
I'm ok w/ it.  I think of it as a Throttling specific set of problems but I can buy into that being a concurrency problem (multiple threads access the hub concurrently causing problems if not adjudicated).